src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTHttpClientHandler.cs

```diff
@@ -0,0 +1,475 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if FEATURE_NETNATIVE
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Security.Cryptography.X509Certificates;
+using System.ServiceModel.Security;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Windows.Security.Cryptography;
+using Windows.Security.Cryptography.Certificates;
+
+using RTApiInformation = Windows.Foundation.Metadata.ApiInformation;
+using RTHttpBaseProtocolFilter = Windows.Web.Http.Filters.HttpBaseProtocolFilter;
+using RTHttpCacheReadBehavior = Windows.Web.Http.Filters.HttpCacheReadBehavior;
+using RTHttpCacheWriteBehavior = Windows.Web.Http.Filters.HttpCacheWriteBehavior;
+using RTHttpCookieUsageBehavior = Windows.Web.Http.Filters.HttpCookieUsageBehavior;
+using RTPasswordCredential = Windows.Security.Credentials.PasswordCredential;
+
+namespace System.ServiceModel.Channels
+{
+    internal class RTHttpClientHandler : DelegatingHandler
```

Comment:
This entire class along with several others will outright disappear. Currently we have ServiceModelHttpMessageHandler which has partial implementations for CoreClr, NetNative and Windows. HttpsChannelFactory in multiple places checks various properties to see if certain capabilities are there before trying to use them. When client cert usage becomes available cross plat, the change will start at HttpsChannelFactory and it will be obvious which code to remove as the classes just won't be referenced any more. I'll open a bug and mention the files which will need to be touched/removed though as we don't know when this will become available.